packages/office-ui-fabric-react/src/components/Modal/Modal.styles.ts

```diff
@@ -67,9 +68,18 @@ export const getStyles = (props: IModalStyleProps): IModalStyles => {
         position: 'relative',
         textAlign: 'left',
         outline: '3px solid transparent',
-        maxHeight: '100%',
+        maxHeight: 'calc(100% - 32px)',
+        maxWidth: 'calc(100% - 32px)',
+        minHeight: '176px',
+        minWidth: '288px',
         overflowY: 'auto',
-        zIndex: isModeless ? ZIndexes.Layer : undefined
+        zIndex: isModeless ? ZIndexes.Layer : undefined,
+        selectors: {
```

Comment:
i'm fine with the other additions, though i could see this media query styling causing issues for modal users, as it's higher specificity, and more difficult to override. Is there a reason these more opinionated styles aren't on the dialog vs modifying this base control?

Reply:
@micahgodbolt Actually, nicely enough, there are styles I put in DialogContent.styles that cover this base. Just spoke with Chris and he said having 16px all around would be fine so I have kept that styling and removed the breakpoints.